app/tailwind.config.js

```diff
@@ -5,11 +5,60 @@ module.exports = {
   darkMode: 'class',
```

Comment:
Dark mode seems to be broken, as it only darkens the button text. I think this is because if you rely on default colors Nightwind doesn't convert those. For example you need to explicitly specify the background color as `bg-white` if you want it to become black (though you probably want use Nightwind overrides to make that a dark gray anyway).

Or maybe we're ok with removing dark mode? In which case we can remove the footer buttons and relevant logic from the settings store

Reply:
yes i asked phil some weeks ago, if we can get rid of darkmode and he agreed.